migrations/versions/9bee3b519bf1_add_history_fingerprint.py

```diff
@@ -0,0 +1,36 @@
+"""add_history_fingerprint
+
+Revision ID: 9bee3b519bf1
+Revises: 167a97c81739
+Create Date: 2023-08-29 19:33:34.654753
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "9bee3b519bf1"
+down_revision = "167a97c81739"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # TODO before I merge this PR: fix this migration code!!
```

Comment:
I marked this PR as draft because I need to fix this migration before we merge. But I'd appreciate some eyes on the PR while I'm on PTO, if you have the time.